mne/viz/utils.py

```diff
@@ -620,7 +622,8 @@ def _mouse_click(event, params):
             return
         params['label_click_fun'](pos)
     # vertical scrollbar changed
-    if event.inaxes == params['ax_vscroll']:
+    if (event.inaxes == params['ax_vscroll'] and
+            'fig_selection' not in params.keys()):
```

Comment:
This could pick the selection whose portion of the scroll bar is under the mouse